### ISSUE TITLE
Crash on Label.Text = "\n" in Debug

### ISSUE THREAD

**Issue by ericoporto** [adventuregamestudio/ags#1993]
**Describe the bug**
After https://github.com/adventuregamestudio/ags/commit/8267528196d4e50c9d6e9f73c4fcee3b56b1ec83, set a label text with a string containing only linefeed causes the game to crash, when debugging in Visual Studio.

This doesn't cause issues in Release but makes it impossible to run games in debug in VS.

**AGS Version**
Current master, since https://github.com/adventuregamestudio/ags/commit/8267528196d4e50c9d6e9f73c4fcee3b56b1ec83

**Game**
[TestLabelCrash.zip](https://github.com/adventuregamestudio/ags/files/11368740/TestLabelCrash.zip)

**To Reproduce**
Run the above game

or create a new game using any template:
1. Create a GUI and adda Label in this GUI.
2. Add the following in Room Script:
```
// room script file

function room_AfterFadeIn()
{
  Label1.Text = "\n";
}
```
3. Build the game
4. Using Visual Studio, debug the engine and pass the above generated .ags file.
5. See the crash happen

**Expected behavior**
This should not cause a crash in the VS debugger

**Screenshots**
![image](https://user-images.githubusercontent.com/2244442/235552556-5f1fc62d-e25f-4cde-9c6e-aac1d32c9a20.png)

**Desktop:**
 - OS: Windows
 - Version 10
 - VS 2019 (Version 16.11.18)

**Comment by ivan-mogilko** [comment]
Pushed a fix: 3fa7e6df41443f368f992aafc80389dc56872e35

**Comment by ericoporto** [comment]
The fix worked! Thanks
